install_config/topics/sssd_for_ldap_failover_prereqs.adoc

```diff
@@ -0,0 +1,40 @@
+////
+sssd for ldap failover and extended attributes prerequisites
+
+Module included in the following assemblies:
+
+* install_config/sssd_for_ldap_failover.adoc
+////
+
+[id='sssd-for-ldap-prereqs-{context}']
+= Prerequisites for configuring basic remote authentication
+
+* Before starting setup, you need to know the following information about your
+LDAP server.
++
+* Whether the directory server is powered by
+http://www.freeipa.org/page/Main_Page[FreeIPA], Active Directory, or another
+LDAP solution.
++
+* The Uniform Resource Identifier (URI) for the LDAP server, for example,
+*ldap.example.com*.
++
+* The location of the CA certificate for the LDAP server.
++
+* Whether the LDAP server corresponds to RFC 2307 or RFC2307bis for user groups.
++
+* Prepare the servers:
++
```

Comment:
The following need retract to make as sbuset, refer https://docs.openshift.com/container-platform/3.4/install_config/advanced_ldap_configuration/sssd_for_ldap_failover.html#sssd-prerequisites-for-authenticating-proxy-setup